src/macho.cpp

```diff
@@ -17,6 +22,10 @@
 
 /**************************************************************************************************/
 
+#if ORC_FEATURE(MACH_O)
```

Comment:
In general, I'd like to minimize the #if FEATURE sort of thing. Can this be turned on? (Even if it is only used by mach-o)

Reply:
I'm not quite sure what you mean by your comment? Ideally what I'd like to do here is not compile ` macho.cpp` for platforms that don't support it, but that would require some CMake changes I'm not entirely comfortable making. Besides, @kc9jud suggests there are tools platforms like Ubuntu can use to convert their object files to Mach-O, in which case we'd only need to transcribe the data structures to headers that are universally available. In _that_ case, this check could go away completely.

Reply:
Sounds good - just making sure it was as minimal as possible.